app/src/main/java/ch/epfl/sdp/social/Conversation/ChatActivity.java

```diff
@@ -45,49 +47,57 @@ protected void onCreate(Bundle savedInstanceState) {
         lv = findViewById(R.id.messages_view);
 
         chattingWith = getIntent().getStringExtra("chattingWith");
-        if(chattingWith == null) {
+        if (chattingWith == null) {
             // instrumentation test running so initialize to sentinel value "null_0"
             chattingWith = "null_0";

```

Comment:
Test code should not be there right? Is there another way to do it?

Reply:
This is again a reformatting. I'm not totally sure of how it's used in the tests, but I'm sure we can set it before the activity launches. I will look at this in another branch, since I want to merge this one.